package.json

```diff
@@ -23,7 +23,7 @@
     "chalk": "1.1.3"
   },
   "peerDependencies": {
-    "stylelint": "^9.0.0"
+    "stylelint": "^9.0.0 || ^10.0.0"
```

Comment:
We should probably just make a breaking change and bump to ^10.0.0.

Reply:
Done 👍